### PR TITLE
Go 1.6.3 complains about unsupported ciphers, removing them

### PR DESCRIPTION
**Pull request by KaptenJansson**
**Description**
Removing unsupported ciphers as they prevent the collider from starting.

Based on tkchin's comment in https://github.com/webrtc/apprtc/issues/272#issuecomment-199419977.

**Purpose**
Making Collider compatible with Go 1.6.3.

### CODE REVIEW

**Comment by KaptenJansson** [comment]
@hkjellander PTAL
cc @tkchin

File: src/collider/collider/collider.go [hunk 1]
```diff
@@ -54,8 +54,6 @@ func (c *Collider) Run(p int, useTls bool) {
 			CipherSuites: []uint16 {
 				tls.TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,
 				tls.TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,
-				tls.TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA,
-				tls.TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA,
 				tls.TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,
 				tls.TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,
 				tls.TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,
```
Comment (hkjellander) [inline src/collider/collider/collider.go]: According to the isBadCipher check at https://github.com/bradfitz/http2/blob/b6255645465a25b25f804acb9b3a54009e80c2a4/server.go#L228-L302 is seems it's better to remove this too.

**Comment by KaptenJansson** [comment]
@hkjellander PTAL

**Comment by hkjellander** [comment]
lgtm

**Pull request merged**